### PR TITLE
RESTWS-681: Document TaskActionResource

### PR DESCRIPTION
**Pull request by Sub6Resources**
I have Documented TaskActionResource as described in https://issues.openmrs.org/browse/RESTWS-681

### CODE REVIEW

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskActionResource1_8.java [hunk 1]
```diff
@@ -181,6 +185,16 @@ public DelegatingResourceDescription getCreatableProperties() throws ResourceDoe
 		return description;
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		ModelImpl model = (ModelImpl) super.getCREATEModel(rep);
+		model.property("tasks", new ArrayProperty());
```
Comment (gayanW) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskActionResource1_8.java]: It is better to make this

    model.property("tasks", new ArrayProperty(new StringProperty()));

Please see.
https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitResource1_9.java#L186

**Comment by Sub6Resources** [comment]
I have committed the changes you requested.

**Comment by gayanW** [comment]
@Sub6Resources Please squash the commits.

**Comment by Sub6Resources** [comment]
The commits have been squashed.

**Comment by dkayiwa** [comment]
@gayanW what do you think of this?

**Comment by gayanW** [comment]
@dkayiwa it looks good.

**Pull request merged**